parsec/backend/backend_events.py

```diff
@@ -0,0 +1,18 @@
+# Parsec Cloud (https://parsec.cloud) Copyright (c) AGPLv3 2019 Scille SAS
+
+from enum import Enum
+
+__all__ = "BackendEvents"
```

Comment:
`__all__` should be a list or a tuple (beside I'm not sure it is needed here, from my point of view `__all__` is only useful in `__init__.py` to re-expose imports)

Reply:
that's a mistake, thanks